lib/sycamore/sycamore/data/table.py

```diff
@@ -0,0 +1,164 @@
+from collections import OrderedDict
+from dataclasses import dataclass, field
+from typing import Any, Optional
+import xml.etree.ElementTree as ET
+
+from sycamore.data import BoundingBox
+from PIL import Image, ImageDraw
+import numpy as np
+from pandas import DataFrame
+
+
+# This data model is similar to that used by Textract and TableTransformers.
+# It is intended to be a common intermediate representation for a variety of
+# table structure recognition models.
+@dataclass(frozen=True)
+class TableCell:
+    content: str
+    rows: list[int]
+    cols: list[int]
```

Comment:
Why are these lists?  Is this for spanning?  Can a list end up disjointed like [1,2,5,6]?

Reply:
It is for spanning. A single cell should not be disjoined. I can try to add some validation logic.